Annotations/Classes/Model/RectModel.swift

```diff
@@ -8,59 +8,83 @@
-    case to
-    case originY
-    case toX
+  case origin
+  case to
+  case originY
+  case toX
 }
 
 let widthDot: Double = 0
 
-public struct RectModel: Model {
-    public let origin: PointModel
-    public let to: PointModel
-    public let color: ModelColor
-    
-    func valueFor(rectPoint: RectPoint) -> PointModel {
-        switch rectPoint {
-        case .origin:
-            return origin.returnPointModel(dx:origin.x + (origin.x < to.x ? widthDot : (-widthDot)), dy:origin.y + (origin.y > to.y ? widthDot : (-widthDot)))
-        case .to:
-            return to.returnPointModel(dx:to.x + (origin.x > to.x ? widthDot : (-widthDot)), dy:to.y + (origin.y > to.y ? widthDot : (-widthDot)))
-        case .originY:
-            return origin.returnPointModel(dx:origin.x + (origin.x < to.x ? widthDot : (-widthDot)), dy:to.y + (origin.y > to.y ? widthDot : (-widthDot)))
-        case .toX:
-            return to.returnPointModel(dx:to.x + (origin.x > to.x ? widthDot : (-widthDot)), dy:origin.y + (origin.y > to.y ? widthDot : (-widthDot)))
-        }
-    }
-    
-    func copyMoving(rectPoint: RectPoint, delta: PointModel) -> RectModel {
-        switch rectPoint {
-        case .origin:
-          return RectModel(origin: origin.copyMoving(delta: delta), to: to, color: color)
-        case .to:
-          return RectModel(origin: origin, to: to.copyMoving(delta: delta), color: color)
-        case .originY:
-          return RectModel(origin: origin.returnPointModel(dx:origin.x + delta.x, dy:origin.y),
-                           to: to.returnPointModel(dx:to.x, dy:to.y + delta.y),
-                           color:color)
-        case .toX:
-          return RectModel(origin: origin.returnPointModel(dx:origin.x, dy:origin.y + delta.y),
-                           to: to.returnPointModel(dx:to.x + delta.x, dy:to.y),
-                           color: color)
-        }
+public class RectModel: Model {
+  public var index: Int
+  public let origin: PointModel
+  public let to: PointModel
+  public let color: ModelColor
+  
+  required init(index: Int, origin: PointModel, to: PointModel, color: ModelColor) {
+    self.index = index
+    self.origin = origin
+    self.to = to
+    self.color = color
+  }
+  
+  func valueFor(rectPoint: RectPoint) -> PointModel {
+    switch rectPoint {
+    case .origin:
+      return origin.returnPointModel(dx:origin.x + (origin.x < to.x ? widthDot : (-widthDot)), dy:origin.y + (origin.y > to.y ? widthDot : (-widthDot)))
+    case .to:
+      return to.returnPointModel(dx:to.x + (origin.x > to.x ? widthDot : (-widthDot)), dy:to.y + (origin.y > to.y ? widthDot : (-widthDot)))
+    case .originY:
+      return origin.returnPointModel(dx:origin.x + (origin.x < to.x ? widthDot : (-widthDot)), dy:to.y + (origin.y > to.y ? widthDot : (-widthDot)))
+    case .toX:
+      return to.returnPointModel(dx:to.x + (origin.x > to.x ? widthDot : (-widthDot)), dy:origin.y + (origin.y > to.y ? widthDot : (-widthDot)))
```

Comment:
@dworak lines seem to be too long here.
In my opinion, it is better to start each parameter from a new line here.
We also need to add a space between ":" and passed value.